client_test.go

```diff
@@ -131,6 +131,134 @@ func (s *clientSuite) TestCall(c *gc.C) {
 	}
 }
 
+var callURLTests = []struct {
```

Comment:
Given that CallURL *is* Call (very nearly), ISTM that these tests are redundant because they're duplicating the tests in TestCall.

I'd suggest one very simple test for CallURL (making sure that the url parameter is actually used) and stay with the TestCall tests.